### PR TITLE
Pinpoint example 

### PR DESCRIPTION
**Pull request by helios1138**
Screenshot of Pinpoint

### CODE REVIEW

**Comment by bfelbo** [comment]
Let's disable `jsx-a11y/alt-text` eslint rule as these warnings are not relevant for this game so `test-docs` succeeds?

**Comment by bfelbo** [comment]
Would be great if you could also update the top-level README to include this example :)

File: multiplayer/examples/pinpoint/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Pinpoint
+
+A multiplayer Pinpoint game using the Rune SDK.
```
Comment (bfelbo) [inline multiplayer/examples/pinpoint/README.md]: Hehe, "Pinpoint game" is not the most descriptive 😅 Let's add more context?

Comment (bfelbo) [reply multiplayer/examples/pinpoint/README.md]: Let's also add a line about the technologies used?

**Review by bfelbo — APPROVED** [review]
Amazing to add this game! 🥳

Would be great to add some comments wherever you think it might be helpful for those interested in how the game code works.

File: multiplayer/examples/pinpoint/README.md (hunk 1 above)
Comment (helios1138) [reply multiplayer/examples/pinpoint/README.md]: done

**Pull request merged**